docs/topics/appendix-glossary.adoc

```diff
@@ -6,7 +6,7 @@
 
 {launcher-oso}:: link:{link-launcher-oso}[{launcher-oso}] is a standalone getting started experience offered by Red Hat for jumpstarting cloud-native application development on OpenShift. It provides a hassle-free way of creating functional example applications, called missions, as well as an easy way to build and deploy those missions to OpenShift. 
 
-{launcher} Service:: The {launcher} service is the upstream project from which link:{link-launcher-oso}[{launcher-oso}] is based.
+{launcher} Tool:: The {launcher} tool is the upstream project from which link:{link-launcher-oso}[{launcher-oso}] is based.
```

Comment:
Neither the "service" nor the "tool" are the upstream project. IMHO, it's just "Fabric8 Launcher is the upstream project".